src/server/graphql/models/IntranetJobs/intranetJobsQuery.js

```diff
@@ -88,12 +89,31 @@ export default {
 
       // RESOLUTION
       const activeThresh = new Date(Date.now() - OLD_MEETING_AGE);
-      const teamIds = await r.table('Meeting')
-        .filter({ endedAt: null }, { default: true })
-        .filter((meeting) => meeting('createdAt').le(activeThresh))('teamId');
-      const promises = teamIds.map((teamId) => endMeeting.resolve(undefined, {teamId}, {authToken, socket: {}}));
+      const idPairs = await r.table('Meeting')
+        .group({index: 'teamId'})                               // for each team
+        .max('createdAt')                                       // get the most recent meeting only
+        .ungroup()('reduction')                                 // return as sequence
+        .filter({ endedAt: null }, { default: true })           // filter to unended meetings
+        .filter(r.row('createdAt').le(activeThresh))('teamId')  // filter to old meetings, return teamIds
+        .do((teamIds) => r.db('actionProduction').table('TeamMember')
+          .getAll(r.args(teamIds), {index: 'teamId'})
+          .filter({isLead: true})
+          .pluck('teamId', 'userId')
+        );                                                       // join by team leader userId
+      const promises = idPairs.map(async ({teamId, userId}) => {
+        await endMeeting.resolve(undefined, {teamId}, {authToken, socket: {}});
```

Comment:
we could make this a thenable to include the next line since 1 isn't dependent on the other, but this is run so infrequently it doesn't really matter